design-guidelines/copy-and-content/date-time-formatting.md

```diff
@@ -0,0 +1,162 @@
+# Date and Time Formatting
+
+Dates and times should be displayed as concisely as possible without preventing the user from seeing the level of detail required to complete their task.
+
+## Referring to dates and times
+
+When writing copy that refers to dates or times, use the same kind of language you would use in written communication.
+
+For example:
+"This cluster will be detached in 2 days."
+"The job ran for 2 hours."
+"This service will be available 01 January 2020."
```

Comment:
This guide is for US English, but this is the European date format. Is that intentional? Same with showing "13:00" and not "1 pm".

Reply:
Yes, that's intentional.